src/ConsentBanner/i18n.tsx

```diff
@@ -0,0 +1,63 @@
+import { createComponentI18nApi } from "../i18n";
+import React from "react";
+import { getLink } from "../link";
+
+export const { useTranslation, addConsentBannerTranslations } = createComponentI18nApi({
+    componentName: "ConsentBanner",
+    frMessages: {
+        "all services pref": "Préférences pour tous les services.",
+        "personal data cookies": "Données personnelles et cookies",
+        "accept all": "Tout accepter",
+        "accept all - title": "Autoriser tous les cookies",
+        "refuse all": "Tout refuser",
+        "refuse all - title": "Refuser tous les cookies",
+        "accept": "Accepter",
+        "refuse": "Refuser",
+        "confirm choices": "Confirmer mes choix",
+        "about cookies": (p: { siteName: string }) => `À propos des cookies sur ${p.siteName}`,
+        "welcome message": (p: { gdprPageLink: string }) => {
+            const { Link } = getLink();
```

Comment:
This is a missuse of the routing mechanim.  
Either we are quarentied that the link is external (it's not the case here) and thus we can use a href and we don't need the `<Link />` component, we can use `<a />` directly.  
Or we don't know if the link is local or external and thus we must acomodate client side rooting and speading `linkProps` on the Link.